PR benchmarks (#36309)
dotnet/aspnetcore: docs/PRBenchmarks.md

# Pull Request Benchmarks

Pull requests that may impact performance should be benchmarked to ensure no regression is introduced.

> The most common way is to build the PR on your development machine and submit the binaries to an existing [scenario](https://github.com/aspnet/Benchmarks/tree/main/scenarios). To simplify this process this repository can do it automatically using GitHub Actions.

Reviewer: ```suggestion
The most common way is to build the PR on your development machine and submit the binaries to an existing [scenario](https://github.com/aspnet/Benchmarks/tree/main/scenarios). For simplicity, this process can be done automatically via GitHub Actions.

```

## Submitting a PR for benchmarking

> A special set of comments are recognized by this repository in order to trigger a benchmark. The comment starts with `/benchmark` and takes the name of the benchmark command as an argument. For instance, `/benchmark kestrel` will build the Kestrel components and submit a `plaintext` scenario benchmark to dedicated machines.

Reviewer: Where does one find the list of benchmarks?

Author: Two lines bellow ;)
